<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workshop</class>
 <widget class="QWidget" name="workshop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>просмотр списка цехов</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/КОД 09.02.07-2-2025 Приложения к образцу задания Том 1 (1)/Ресурсы/Мастер пол.ico</normaloff>../../Downloads/КОД 09.02.07-2-2025 Приложения к образцу задания Том 1 (1)/Ресурсы/Мастер пол.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;просмотр списка&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;цехов&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_workshop">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>461</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4E8D3 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back_worlshop">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #67BA80 </string>
   </property>
   <property name="text">
    <string>назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>наименование цеха</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_products_workshop">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
